scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -303,7 +304,9 @@
     (async/chan 1))
 
   (pending-offers [this pool-name]
-    (let [pods (add-starting-pods this @all-pods-atom)
+    (log/info "In" name "compute cluster, looking for offers for pool" pool-name)
+    (let [timer (timers/start (metrics/timer "cc-k8s-pending-offers-compute" name))
```

Comment:
Can we stay consistent with the other timers in this namespace (e.g. `cc-kill-tasks`) and just do:

```suggestion
    (let [timer (timers/start (metrics/timer "cc-pending-offers-compute" name))
```